fetch/api/request/request-cache.js

```diff
@@ -75,7 +79,7 @@ function make_url(uuid, id, value, content, info) {
     "&content=" + content +
     "&" + id + "=" + value +
     "&expires=" + dates[info.state] +
-    vary + cache_control + ignore_request_headers;
+    vary + cache_control + pragma + ignore_request_headers;
```

Comment:
It is not clear to me what you are trying to do here with Pragma.
cache.py is grabbing the Pragma request header and storing it in its state so that it can later be validated by the JS test. Adding it in the URL should not change anything.
We are adding Cache-Control in the URL so that the response will contain Cache-Control header value.

Reply:
It was my understanding that to get a response to be consistently ignored by browser caches you needed to set both of these headers:

```
cache-control: no-store
pragma: no-cache
```

This diff hunk was about getting the pragma header set on the response toward this end.

Reply:
Oh, I see.  I saw that pragma was being read from the request in `cache.py` and I guess I thought it was reflected back.

Since the test is adequate to cover the situation I'm trying to test without the pragma stuff actually working, then I guess I'll just remove it.  Looking closer, it seems its fine for a server to simply return `cache-control: no-store` in the response header.